.ci/snap_publish.sh

```
@@ -0,0 +1,52 @@
+#!/bin/bash
+set -euo pipefail
+
+SNAPCRAFT_BUILD_ENVIRONMENT_CPU=$(nproc)
+export SNAPCRAFT_BUILD_ENVIRONMENT_CPU
+SNAPCRAFT_BUILD_ENVIRONMENT_MEMORY=16G
+export SNAPCRAFT_BUILD_ENVIRONMENT_MEMORY
```

```
export SNAPCRAFT_BUILD_ENVIRONMENT_CPU=$(nproc)
export SNAPCRAFT_BUILD_ENVIRONMENT_MEMORY=16G
```
?

---

And the same for `THREADS` and `ROOT`

---

Shell check suggestion. Export+assign+command makes it fail silently.

---

Woops.. that means memory could have gone on a single line.

---

I had no idea.

---

Yeah, me neither!